theseus/optimizer/tests/test_manifold_gaussian.py

```diff
@@ -0,0 +1,196 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import copy
+
+import numpy as np
+import pytest  # noqa: F401
+import torch
+
+import theseus as th
+from theseus.optimizer.manifold_gaussian import local_gaussian, retract_gaussian
```

Comment:
We should also add these to the appropriate init files if we are not moving them into the ManifoldGaussian class. If we are then it would makes sense to just call then local and retract? Unfortunately we can't rename the external ones since the Manifold class already has a local/retract in the `th.` namespace.

Reply:
Maybe in a later PR we could rename the old `local` and `retract` to something like `_local_lie_group`, and these ones to something like `_local_gaussian`. Then in Theseus' `__init__.py` we define `local` to route to the appropriate one based on the type.

Reply:
Nice suggestion! Added issue #159.